WebApp/WebApp/Features/Student/Dashboard/Dashboard.cshtml

```diff
@@ -1,16 +1,109 @@
-﻿<div class="page-content">
+﻿@model WebApp.Features.Student.Dashboard.DashboardController
```

Comment:
So, you actually don't have to pass the Controller object as a model back to the cshtml file. (I'm surprised it actually let you do this, but there aren't any errors being thrown). If a model isn't needed on the page, you can just not include the @model line.

Reply:
Makes sense. It has been updated